src/vs/platform/electron/electron-main/electronMainService.ts

```diff
@@ -366,15 +366,13 @@ export class ElectronMainService implements IElectronMainService {
 	//#region Connectivity
 
 	async resolveProxy(windowId: number | undefined, url: string): Promise<string | undefined> {
```

Comment:
(maybe I am missing smth, but still..) Is current version of return type still valid after change is is below?